### PR TITLE
docs: add a note about running migrations after changing database type

### PR DESCRIPTION
**Pull request by jonjahr**
I think it will be helpful to remind newbies to run migrations after they change their `database_type`.  I'm a newbie and I got stuck for a while today because I didn't know I had to run migrations after I changed from `sqlite` to `postgres`.

### CODE REVIEW

File: docs/content/usage/configurations.md [hunk 1]
```diff
@@ -80,6 +80,12 @@ DATABASE_URL=<YOUR_DATABASE_URL>
 
 Where `<YOUR_DATABASE_URL>` is the URL of your PostgreSQL database.
 
+Remember to run migrations after you change your database type to `postgres` from another type:
+
+```bash
+medusa migrations run
+```
+
```
Comment (shahednasser) [inline docs/content/usage/configurations.md:88]: Can you add a new H3 header before this called "Changing Database Type" to make it easier to find?

**Review by shahednasser — CHANGES_REQUESTED** [review]
Thank you for your contribution @jonjahr ! Just a small comment below

**Comment by shahednasser** [comment]
@jonjahr are you still working on this?

**Comment by jonjahr** [comment]
Oh sorry, thanks for the reminder!  Yes I want to do this, I’ll do this today.
________________________________

File: docs/content/usage/configurations.md (hunk 1 above)
Comment (jonjahr) [reply docs/content/usage/configurations.md:90]: @shahednasser Here you go, I've added the H3 header.

**Review by shahednasser — APPROVED** [review]
Thanks for making the change!

**Comment by shahednasser** [comment]
Great thank you! @jonjahr

**Pull request merged**